release-payloads/release-1.2.1/release-stage-bundle.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Release
+metadata:
+  namespace: cluster-observabilit-tenant
+  name: coo-1.2.1-stage-release-x97hm-2025-06-25-17-44
```

Comment:
@jan--f I was not sure if this name is random plus the date, or better to use the snapshot suffix + date

Reply:
nvm just realized the script does it this way.